cpp/tests/round/round_tests.cpp

```diff
@@ -326,14 +322,15 @@ TYPED_TEST(RoundTestsFixedPointTypes, TestScaleMovementExceedingMaxPrecision)
   auto const input =
     fp_wrapper{{14, 15, 16, 24, 25, 26, -14, -15, -16, -24, -25, -26}, scale_type{1}};
   auto const expected = fp_wrapper{{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}, scale_type{target_scale}};
-  auto const result   = cudf::round(input, -target_scale, cudf::rounding_method::HALF_UP);
```

Comment:
since the round method is only marked as deprecated in the meantime, why don't we preserve the tests as well? and remove the test for it once we remove it